spec/services/contacts_service_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+RSpec.describe ContactsService do
+  describe "#by_content_id" do
+    context "when a contact is found" do
+      it "returns the contact" do
+        content_id = SecureRandom.uuid
```

Comment:
It's nice to see the full attributes of a contact, but it took me a while to see that most of it isn't necessary for the test, but it's presence kind of suggested it was. Could we keep the test data to a minimum so it's easier to see the data under test?